dist/output.css

```diff
@@ -585,6 +585,31 @@ select {
   outline: 1px auto -webkit-focus-ring-color;
 }
 
+.message_bubble {
+  width: -moz-fit-content;
+  width: fit-content;
+  max-width: 80%;
```

Comment:
can you add max-height too, for bubbles that have very long and thin image messages?

Reply:
sorry could you provide an example of a long and thin image message? What should the max height be set to?

Reply:
e.g. screenshot of my phone

Reply:
let's say max-height should be 50% of the layout viewport?

Reply:
dont worry about it if too bothersome